Dockerfile.dist

```diff
@@ -17,8 +17,8 @@
 ## Build
 
 # Ideally, use the official image from Red Hat, e.g. registry.access.redhat.com/ubi9/go-toolset,
-# but a 1.22 release does not yet exist.
-FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.22.7@sha256:376dd8d1291580a32824c7a072b72e1dce524bb9d300393b656931ba6156b86d AS build
+# but a 1.23 release does not yet exist.
```

Comment:
Is this still true?

Reply:
This is the latest I can find. https://catalog.redhat.com/software/containers/ubi9/go-toolset/61e5c00b4ec9945c18787690

Reply:
Looks like yes:

```
podman pull -q registry.redhat.io/ubi9/go-toolset:latest && podman run --rm -it --entrypoint /bin/bash registry.redhat.io/ubi9/go-toolset:latest -c "go version"
91162a41a03a99186ff6ffab0f79966fab706971a933b2b4d80055bf1e35a70e
go version go1.22.9 (Red Hat 1.22.9-2.el9_5) linux/amd64
```